ajax/libs/just-animate/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "just-animate",
-  "filename": "just-animate-animations.min.js",
+  "filename": "just-animate-all.min.js",
```

Comment:
Please don't touch filename here unless you added a version with the new name. I can take care of this change, thanks.